tests/Microsoft.CodeAnalysis.Testing/Directory.Build.props

```diff
@@ -21,7 +21,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
 
   <PropertyGroup>
-    <TestTargetFrameworks>netcoreapp3.1;net472;net46</TestTargetFrameworks>
```

Comment:
forcing the `Newtonsoft.Json` version means that net46 will find the wrong assemblies at runtime in this repo. We will get flagged if we have tests that use a `Newtonsoft.Json` version that is older than `13.0.1`. I don't have any good ideas on how to solve this problem other than to remove the TFM from our test suite.